### PR TITLE
Add Meetings API examples

### PR DESCRIPTION
**Pull request by S-H-GAMELINKS**
Hi.

I added Meetings API example in `__example/meetings`.
I'd be glad to merge these example.

### CODE REVIEW

**Comment by S-H-GAMELINKS** [comment]
@rafecolton
Hi.
How is this pull request?

File: _example/meetings/delete_meeting.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+		log.Printf("Error: %+v\n\n", err)
+	}
+
+	deleteMeetingOpt := zoom.DeleteMeetingOptions{
+		MeetingID: id,
+	}
+
+	err = zoom.DeleteMeeting(deleteMeetingOpt)
+
+	if err != nil {
```
Comment (rafecolton) [inline _example/meetings/delete_meeting.go]: Please prefer abbreviated syntax here:

```suggestion
	if err = zoom.DeleteMeeting(deleteMeetingOpt); err != nil {
```

**Review by rafecolton — CHANGES_REQUESTED** [review]
Requesting one small change, everything else LGTM

**Comment by S-H-GAMELINKS** [comment]
@rafecolton 
Hi.
Thanks for your review comment!

Fixed it in https://github.com/zoom-lib-golang/zoom-lib-golang/pull/32/commits/54bd9af1d412ef0bdb0767ce2ff458e8c7c526bf

**Review by rafecolton — APPROVED** [review]

**Pull request merged**